pyodide-build/pyodide_build/pypabuild.py

```diff
@@ -186,7 +186,7 @@ def get_build_env(
     cxxflags: str,
     ldflags: str,
     target_install_dir: str,
-    exports: _BuildSpecExports | list[_BuildSpecExports],
+    exports: _BuildSpecExports | list[str],
```

Comment:
Maybe we can define a type for `_BuildSpecExports | list[str]`.  It is used everywhere in this PR.